### DISCUSSION TITLE
2024-02-13-my-2023

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#169] [General]
# 2023年总结 - 随心而行 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life今天是大年初四，哈哈我的年度总结又拖延了，2022 年总结「率性而活」也是这个时候，这一年根据自己喜好做了很多想做的事情，小标题就叫「随心而行」吧，打算通过自问自答的方式来聊聊我 2023 年的有趣生活、折腾、工作。

https://tw93.fun/2024-02-13/my-2023.html

**Comment by baifengbai** [comment]
总结下就是有了apple的新产品，生活和工作更幸福了

**Reply by tw93** [reply]
也不是，🤣 应该没有认真看

**Comment by QuakeWang** [comment]
新的一年，多思考、多读书、多锻炼。有时候写自己感兴趣的代码真的是一种很好的解压方式 哈哈～～🎉

**Reply by tw93** [reply]
是的是的

**Comment by liucoffe2** [comment]
勤奋的人总是被财神爷青眼有加。

**Reply by tw93** [reply]
谢谢，可能这个叫做长期主义

**Comment by ChevalierWang** [comment]
请问博主有调研过近视手术的术后风险和后遗症吗？ 我也一直挺想做的，又担心有后遗症

**Reply by tw93** [reply]
有的，不过这个建议去正规的医院，多问问医生

**Comment by Trade-Offf** [comment]
看完了很羡慕，觉得你生活工作的环境非常正向积极。但我想原因可能出自你本身，就是一个强大的正反馈机制，能不断地为自己和社群带来成长和繁荣。祝越来越好 👍🏻

**Reply by tw93** [reply]
谢谢，你也是！

**Comment by yyt520** [comment]
关注大佬很久了，真羡慕大佬的work life balance

**Reply by tw93** [reply]
哈哈其实也不 balance，属于挤时间生活的感觉
